coding-concepts.js

```diff
@@ -8,39 +8,45 @@
 // --------------------1) What will this log?
 
 var cohort = "Delta 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Delta, ,2022
+// b) Verify and explain: (Answer/Explanation found below:)
+
+'D', 'e', 'l', 't',
+'a', ' ', '2', '0',
+'2', '2'
+
+//Explain: .split is a built-in method that turns strings into array. Often times it can used with its "opposite method" .join which conversely turns arrays into strings. 
 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: undefined 
+// b) Verify and explain: undefined - This was undefined because "name" was never specified previously so there is no context for what used, it must be defined first before it came come back with a sufficient output.  
```

Comment:
it is actually missing the return statement.  I do get your confusion though.  "LEARN Student" is actually the name that is being passed in.  name is the parameter (a placeholder) and we pass "LEARN Student" into the greeter method as an argument ----  greeter("LEARN Student").  You show that you know a variable that has not been assigned a value would console.log Undefined.